src/pextlib1.0/tracelib.c

```diff
@@ -614,10 +616,48 @@ static void dep_check(int sock, char *path) {
         answer(sock, "#");
     }
 
+    if (-1 == fs_cs) {
```

Comment:
This may differ per path. Some people may have random mount points within `/opt/local`, so it's probably not a very good idea to cache this.